### PR TITLE
SPCTR-2574 - New: Common NavBar Structure

### PR DESCRIPTION
**Pull request by bsf-zanev**
### Description
Added a Common Multi-purpose Navbar Structure to Spectra's Admin Dashboard.

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
Tested as stated in [this comment](https://brainstormforce.atlassian.net/browse/SPCTR-2574?focusedCommentId=36601).

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by aapatre — APPROVED** [review]

File: admin-core/assets/src/common/components/AdminHeader.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+					<div className='flex-1 flex items-center justify-center sm:items-stretch sm:justify-start'>
+						{ iconLogo ? (
+							<>
+								<div className='flex-shrink-0 flex items-center'>
+									<img
+										className={ uagbClassNames( [
+											'h-10',
+											title && 'mr-6',
+										] ) }
+										src={ uag_react.logo_icon_url }
```
Comment (adi3890) [inline admin-core/assets/src/common/components/AdminHeader.js:28]:
```suggestion
										src={ uag_react?.logo_icon_url }
```
Please verify if we should have these check.
also check for other localized data used.

File: admin-core/inc/admin-menu.php [hunk 2]
```diff
@@ -234,6 +234,7 @@ public function styles_scripts() {
 				'uag_base_url'             => admin_url( 'admin.php?page=' . $this->menu_slug ),
 				'plugin_dir'               => UAGB_URL,
 				'plugin_ver'               => UAGB_VER,
+				'logo_icon_url'            => UAGB_URL . 'admin-core/assets/images/uag-logo.svg',
```
Comment (adi3890) [inline admin-core/inc/admin-menu.php:237]: We already have plugin_dir localized why do we need to localize mre data.
I think we should localize only variable art static part can be used in react as per requirement.

Please verify if we can remove logo_icon_url, logo_url from localized data

**Review by adi3890 — APPROVED** [review]

File: admin-core/assets/src/common/components/AdminHeader.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```
Comment (AbhijitNage123) [inline admin-core/assets/src/common/components/AdminHeader.js:1]: Can we remove it?

File: admin-core/inc/admin-menu.php (hunk 2 above)
Comment (bsf-zanev) [reply admin-core/inc/admin-menu.php]: I've safely removed the localization of the image paths.

**Review by AbhijitNage123 — APPROVED** [review]

**Review by rahulvarma722 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**